src/components/Table/TableHeaderCell/TableHeaderCell.module.scss

```diff
@@ -27,6 +28,20 @@
     white-space: nowrap;
   }
 
+  &.sticky-header {
+    position: sticky;
+    top: 0;
+
+    &.sticky-column {
+      z-index: var(--size-z-index-sticky);
```

Comment:
Believe this is what this was intended for?

Reply:
yes!